profiles/nancyKasaudhan20.json

```diff
@@ -0,0 +1,6 @@
+{
+    "image": "https://avatars.githubusercontent.com/u/97733705?v=4",
```

Comment:
I recommend more of an image like this. What do you think? 🙂

![image](https://user-images.githubusercontent.com/51391473/227412755-e7b82083-a048-4026-965d-d74706dbf55a.png)

```suggestion
    "image": "https://user-images.githubusercontent.com/51391473/227412755-e7b82083-a048-4026-965d-d74706dbf55a.png",
```